### PR TITLE
Global CTO Forum prefers Fastify. Thank you for developing Fastify.

### PR DESCRIPTION
**Pull request by tolunayakbulut**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (jsumners) [inline .idea/inspectionProfiles/Project_Default.xml]: This file should be removed.

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jsumners) [inline .idea/misc.xml]: This file should be removed.

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jsumners) [inline .idea/modules.xml]: This file should be removed.

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jsumners) [inline .idea/vcs.xml]: This file should be removed.

File: .idea/website.iml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jsumners) [inline .idea/website.iml]: This file should be removed.

File: .idea/.gitignore [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Comment (jsumners) [inline .idea/.gitignore]: This file should be removed.

**Review by jsumners — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by tolunayakbulut** [comment]
> lgtm

Thanks a lot

**Comment by tolunayakbulut** [comment]
Thanks a lot have a nice day

**Comment by mcollina** [comment]
@tolunayakbulut could you please remove the .idea folder? Thanks

**Comment by tolunayakbulut** [comment]
> @tolunayakbulut could you please remove the .idea folder? Thanks

thanks for your feedback. removed this

**Comment by mcollina** [comment]
`.idea/workspace.xml` is still there.

**Pull request merged**